1668064800113	Sync.LogManager	DEBUG	Flushing file log
1668064800117	Sync.LogManager	DEBUG	Log cleanup threshold time: 1667200800117
1668064800119	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1668064800119	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1668064800119	Sync.Service	DEBUG	User-Agent: Firefox/89.0.2 (Windows NT 10.0; Win64; x64) FxSync/1.91.0.20210622155641.desktop
1668064800119	Sync.Service	INFO	Starting sync at 2022-11-10 08:20:00 in browser session G5ioxZZftwEB
1668064800119	Sync.Service	DEBUG	In sync: should login.
1668064800119	Sync.Service	INFO	User logged in successfully - verifying login.
1668064800120	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1668064800120	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1668064800120	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1668064800120	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1668064800120	Sync.ErrorHandler	ERROR	Sync encountered a login error
1668064800120	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1668064800120	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1668064800120	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.js:1038:15) JS Stack trace: onNotify@service.js:1038:15
1668064800120	Sync.Service	DEBUG	Not syncing: login returned false.
